internal/api/v1beta1/job_types.go

```diff
@@ -68,6 +79,8 @@ type JobList struct {
 // Policy represents the policy types a job can have
 type Policy struct {
 	RestartPolicy RestartPolicy `json:"restartPolicy,omitempty"`
+	// CronJob-specific
```

Comment:
what do you think if we use RestartPolicy and ConccurentPolicy defined in go-client?

```golang

import (
	batchv1 "k8s.io/api/batch/v1"
	corev1 "k8s.io/api/core/v1"
)

// Policy represents the policy types a job can have
type Policy struct {
	RestartPolicy corev1.RestartPolicy `json:"restartPolicy,omitempty"`
	// CronJob-specific
	ConcurrencyPolicy batchv1.ConcurrencyPolicy `json:"concurrencyPolicy,omitempty"`
}
```